routes/campaign.php

```diff
@@ -35,6 +35,10 @@
 Route::post('/maps/{map}/groups/bulk', 'Maps\MapGroupController@bulk')->name('maps.groups.bulk');
 Route::post('/maps/groups/reorder', 'Maps\MapGroupController@reorder')->name('maps.groups.reorder-save');
 
+Route::get('/maps/{map}/layers', 'Maps\MapLayerController@layers')->name('maps.layers');
+Route::post('/maps/{map}/layers/bulk', 'Maps\MapLayerController@bulk')->name('maps.layers.bulk');
+Route::post('/maps/layers/reorder', 'Maps\MapLayerController@reorder')->name('maps.layers.reorder-save');
```

Comment:
I'm confused why `{map}` isn't included here. Should be, for permissions, as well as for the group reorder.